source/getting-started.rst

```diff
@@ -4,20 +4,21 @@
 Getting Started
 ###############
 
-Sandbox environment
-===================
+This page is intented to familiarize you with superficial parts of the 3-D
+Secure v2 and to provide you with *some* understanding of how a 3-D Secure v2
+authentication flow looks.
 
-A sandbox environment is included to ease integration. It is
-provided as a service for continuous integration and for live tests.
-This is our own implementation so there will be some discrepancies with the
-production endpoint. The production endpoint is to be used only for production requests.
+Authentication
+==============
 
-.. warning::
-  Under *no* circumstances may real card numbers or other cardholder
-  information be sent to the sandbox.
+In 3-D Secure v2, cardholder authentication might not necessarily involve the
+cardholder.
 
-Authentication scenarios
-========================
+A authentication flow that does not involve the cardholder is called a
+*frictionless flow*, an flow that involves the cardholder is called a
```

Comment:
`s/, an/, whereas a/`

Reply:
Nah. Perhaps ", whereas a flow involving the cardholder".

Reply:
I like it :+1: